### PR TITLE
chore(templates):  change`&&` into `npm-run-all`

### PR DESCRIPTION
**Pull request by wxh06**
- [x] Templates

Change `remix build && run-p "dev:*"` into `npm-run-all build -p "dev:*"`.
As a result, it's no longer necessary to edit multiple `scripts` in `package.json` when modifying `build` script.

For example, adding `rsync` to copy fonts required by CSS `@font-face`. See https://github.com/remix-run/remix/issues/1153#issuecomment-1056989834

```json
{
  "scripts": {
    "build": "remix build && rsync -r node_modules/bootstrap-icons/font/fonts public/build/_assets",
    "dev": "npm-run-all build -p \"dev:*\"",
  }
}
```

Reference: https://github.com/mysticatea/npm-run-all/blob/master/docs/npm-run-all.md#run-a-mix-of-sequential-and-parallel-scripts

### CODE REVIEW

**Review by machour — APPROVED** [review]

**Comment by chaance** [comment]
@wxh06 Mind resolving conflicts so I can get this merged?

File: templates/arc/package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "build": "remix build",
     "dev:remix": "remix watch",
     "dev:arc": "cross-env NODE_ENV=development arc sandbox",
-    "dev": "remix build && run-p \"dev:*\"",
+    "dev": "npm-run-all build -p \"dev:*\"",
```
Comment (MichaelDeBoey) [inline templates/arc/package.json:8]:
```suggestion
    "dev": "npm-run-all build --parallel \"dev:*\"",
```

File: templates/cloudflare-pages/package.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
     "build": "remix build",
     "dev:remix": "remix watch",
     "dev:wrangler": "cross-env NODE_ENV=development npm run wrangler",
-    "dev": "remix build && run-p \"dev:*\"",
+    "dev": "npm-run-all build -p \"dev:*\"",
```
Comment (MichaelDeBoey) [inline templates/cloudflare-pages/package.json:8]:
```suggestion
    "dev": "npm-run-all build --parallel \"dev:*\"",
```

File: templates/cloudflare-workers/package.json [hunk 3]
```diff
@@ -6,7 +6,7 @@
     "deploy": "wrangler publish",
     "dev:remix": "remix watch",
     "dev:miniflare": "cross-env NODE_ENV=development miniflare ./build/index.js --watch",
-    "dev": "remix build && run-p \"dev:*\"",
+    "dev": "npm-run-all build -p \"dev:*\"",
```
Comment (MichaelDeBoey) [inline templates/cloudflare-workers/package.json:9]:
```suggestion
    "dev": "npm-run-all build --parallel \"dev:*\"",
```

File: templates/deno/package.json [hunk 4]
```diff
@@ -4,7 +4,7 @@
   "scripts": {
     "build": "remix build",
     "deploy": "deployctl deploy --prod --include=build,public --project=<your deno deploy project> ./build/index.js",
-    "dev": "remix build && run-p \"dev:*\"",
+    "dev": "npm-run-all build -p \"dev:*\"",
```
Comment (MichaelDeBoey) [inline templates/deno/package.json:7]:
```suggestion
    "dev": "npm-run-all build --parallel \"dev:*\"",
```

File: templates/express/package.json [hunk 5]
```diff
@@ -3,7 +3,7 @@
   "sideEffects": false,
   "scripts": {
     "build": "remix build",
-    "dev": "remix build && run-p \"dev:*\"",
+    "dev": "npm-run-all build -p \"dev:*\"",
```
Comment (MichaelDeBoey) [inline templates/express/package.json:6]:
```suggestion
    "dev": "npm-run-all build --parallel \"dev:*\"",
```

**Review by MichaelDeBoey — CHANGES_REQUESTED** [review]
Maybe we should make the parallel part a bit more explicit now? 🤔

**Review by MichaelDeBoey — APPROVED** [review]

**Pull request merged**